packages/office-ui-fabric-react/package.json

```diff
@@ -69,6 +68,8 @@
     "@fluentui/react-focus": "^7.12.25",
     "@fluentui/react-icons": "^0.1.40",
     "@microsoft/load-themed-styles": "^1.10.26",
+    "@fluentui/react-conformance": "^0.1.0",
+    "@fluentui/react-component-ref": "^0.50.0",
```

Comment:
Remove the dep on react-component-ref and move react-conformance back to devDependencies (test-only stuff should never be in dependencies). If that makes a lint rule complain we’ll fix the lint rule config.